build/webpack.common.config.js

```diff
@@ -0,0 +1,136 @@
+'use strict';
```

Comment:
needs adding `build/` to gitignore? All n- use `public/`, it'd be good to keep a similar structure